src/components/TooltipNew/UserGuideTooltip.js

```diff
@@ -127,6 +128,7 @@ class UserGuideTooltip extends React.PureComponent {
           gap={rect}
           isVisible={isVisible}
           slide={shouldSlide}
+          disableClicks={disableSpotlightPointerEvents}
```

Comment:
can you also use `disableSpotlightPointerEvents` property name here?

Reply:
ok